src/lib_c/x86_64-windows-msvc/c/int_safe.cr

```diff
@@ -0,0 +1,3 @@
+lib LibC
+  alias DWord = UInt32
```

Comment:
Isn't this `DWORD`?

Reply:
True. I was just copying that from `winapi.cr`. Changing that will require to change it in that file as well. Not sure if that should be in this PR... WDYT?

Reply:
Ugh, leave it as-is for now and i'll send a PR to rip out `winapi.cr` and redo that to the "new" specs.